packages/mobile/src/screens/sign-on-screen/components/layout.tsx

```diff
@@ -118,20 +118,20 @@ export const Heading = (props: HeadingProps) => {
 
 type ReadOnlyFieldProps = {
   label: string
-  value: string
+  value: string | ReactNode
 }
 
 export const ReadOnlyField = (props: ReadOnlyFieldProps) => {
   const { label, value } = props
 
   return (
-    <Box>
+    <Flex gap='xs'>
```

Comment:
there's no gap in the figma but for some reason the text is way too squished together 🤔 
Adding a gap here seems to fix it